frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -84,11 +85,11 @@ export enum ListFilterId {
 }
 
 export const ListFilterLabels = {
-  [ListFilterId.Running]: 'Running',
-  [ListFilterId.Failed]: 'Failed',
-  [ListFilterId.Succeeded]: 'Succeeded',
-  [ListFilterId.Cancelled]: 'Cancelled',
-  [ListFilterId.Other]: 'Other',
+  [ListFilterId.Running]: i18next.t('public~Running'),
+  [ListFilterId.Failed]: i18next.t('public~Failed'),
+  [ListFilterId.Succeeded]: i18next.t('public~Succeeded'),
+  [ListFilterId.Cancelled]: i18next.t('public~Cancelled'),
+  [ListFilterId.Other]: i18next.t('public~Other'),
```

Comment:
ns for the pipelines-plugin package is `pipelines-plugin`. 
```suggestion
  [ListFilterId.Running]: i18next.t('pipelines-plugin~Running'),
  [ListFilterId.Failed]: i18next.t('pipelines-plugin~Failed'),
  [ListFilterId.Succeeded]: i18next.t('pipelines-plugin~Succeeded'),
  [ListFilterId.Cancelled]: i18next.t('pipelines-plugin~Cancelled'),
  [ListFilterId.Other]: i18next.t('pipelines-plugin~Other'),
```

Reply:
modified the changes and added translation for pods as well.